metadata-ingestion/src/datahub/integrations/great_expectations/action.py

```diff
@@ -1,10 +1,10 @@
-import json
 import logging
 import time
 from dataclasses import dataclass
 from datetime import timezone
 from typing import Any, Dict, List, Optional, Union
 
+import simplejson as json
```

Comment:
why do we need simplejson here?

Reply:
`simplejson.dumps` can serialize `Decimal` type, whereas `json.dumps`

Reply:
Let me know if use of simplejson is too much to solve this. There are alternate solutions that may work, but give less ideal serialization. The other approach is that we just skip the data that standard library(json) fails to serialize.

Reply:
Let's try to avoid introducing this dep if possible. Can we do something like suggested here: [Python JSON serialize a Decimal object](https://stackoverflow.com/questions/1960516/python-json-serialize-a-decimal-object)?

Reply:
yes, we can do that. Its not ideal but not that bad either. I'll make the changes.